scripts/cloud-api-demo/correct_permissions.py

```diff
@@ -0,0 +1,29 @@
+import argparse
+import subprocess
+
+def execute_command(cmd):
+    """Execute a shell command."""
+    process = subprocess.run(cmd, capture_output=True, text=True)
+
+    if process.returncode != 0:
+        print(f"Error occurred: {process.stderr}")
```

Comment:
If there is an error it must immediately exit with a non zero exit code after printing the message

Reply:
Implemented